### PR TITLE
further interesting genome test cases

### PR DESCRIPTION
**Pull request by realmarcin**

### CODE REVIEW

**Comment by dangunter** [comment]
now you have letters and numbers, but you only need one sort prefix. 

06_hC_elegans_chr1_transsplicing -> 06_C_elegans_chr1_transsplicing

**Comment by realmarcin** [comment]
sure, just did a new PR

**Comment by dangunter** [comment]
you should just comment in existing PR

otherwise history is hard to follow

**Comment by dangunter** [comment]
Looks good now. Closing
